src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,145 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+import type { HTMLProps } from "react";
+
+export type Props = {
+  className?: string;
```

Comment:
It's a good practice to allow `null` (`className?: string | null`;) for optional props in case the value is derived from another value or from a function return.

Reply:
Oh, I thought that making a prop optional does exactly that by default.

Reply:
I checked if you can assign `null` if prop is optional, and it seems to be working fine without explicitly adding `null` to type:

<img width="213" alt="Screenshot 2021-02-09 at 08 07 20" src="https://user-images.githubusercontent.com/83575/107327769-f9eedf00-6aad-11eb-9403-8eb3ef4d67d5.png">
<img width="227" alt="Screenshot 2021-02-09 at 08 06 59" src="https://user-images.githubusercontent.com/83575/107327775-fb200c00-6aad-11eb-8ae7-254ddda75227.png">
<img width="221" alt="Screenshot 2021-02-09 at 08 06 45" src="https://user-images.githubusercontent.com/83575/107327777-fbb8a280-6aad-11eb-9df2-b5bfc4939f8a.png">
<img width="201" alt="Screenshot 2021-02-09 at 08 07 11" src="https://user-images.githubusercontent.com/83575/107327774-fb200c00-6aad-11eb-8271-49fe44439ecf.png">